kufar/books/main.go

```diff
@@ -439,25 +460,20 @@ func (d Database) UpdateBook(w http.ResponseWriter, r *http.Request) {
 		return
 	}
 
+	//dont allow bk.UserID to be changed in request body
 	var bk Book
 	if err := json.NewDecoder(r.Body).Decode(&bk); err != nil {
```

Comment:
problem is here. I can edit the userID in my request body, and then change the books of other users